test/extensions/filters/udp/udp_proxy/udp_proxy_filter_test.cc

```diff
@@ -578,19 +578,19 @@ stat_prefix: foo
   test_sessions_[0].expectWriteToUpstream("hello", 0, nullptr, true);
   recvDataFromDownstream("10.0.0.1:1000", "10.0.0.2:80", "hello");
   checkTransferStats(5 /*rx_bytes*/, 1 /*rx_datagrams*/, 0 /*tx_bytes*/, 0 /*tx_datagrams*/);
-  EXPECT_EQ(5, factory_context_.cluster_manager_.thread_local_cluster_.cluster_.info_->stats_
-                   .upstream_cx_tx_bytes_total_.value());
+  EXPECT_EQ(5, factory_context_.cluster_manager_.thread_local_cluster_.cluster_.info_
+                   ->traffic_stats_->upstream_cx_tx_bytes_total_.value());
```

Comment:
I'm surprised to see this particular diff; I thought you already partitioned the stat into separate blocks in a previous PR.

Reply:
the renaming? it's not included in the previous PRs.